tests/system/src/test/java/test/robot/javafx/scene/DatePickerUpdateOnAlertCloseTest.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package test.robot.javafx.scene;
+
+import java.util.concurrent.CountDownLatch;
+import java.time.LocalDate;
+
+import javafx.application.Application;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Alert.AlertType;
+import javafx.scene.control.Button;
+import javafx.scene.control.DatePicker;
+import javafx.scene.input.MouseButton;
+import javafx.application.Platform;
+import javafx.scene.robot.Robot;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import test.util.Util;
+
+/*
+ * Test for verifying DatePicker update on closing the Alert dialog.
+ *
+ * There is 1 test in this file.
+ * Steps for testDatePickerUpdateOnAlertClose()
+ * 1. Create a alert dialog and add date picker to it.
+ * 2. Add button to scene and show alert dialog on button click.
+ * 3. Click on date picker and select a date from popup.
+ * 4. Verify that selected date is updated in the date picker.
+ */
+
```

Comment:
minor: unnecessary newline.

otherwise, very solid code.  thank you Karthik!

Reply:
Removed new line. Thanks Andy.